lms/djangoapps/instructor/views/api.py

```diff
@@ -2017,6 +2017,29 @@ def export_ora2_data(request, course_id):
     return JsonResponse({"status": success_status})
 
 
+@transaction.non_atomic_requests
+@require_POST
+@ensure_csrf_cookie
+@cache_control(no_cache=True, no_store=True, must_revalidate=True)
+@require_course_permission(permissions.CAN_RESEARCH)
+@common_exceptions_400
+def extract_ora2_submission_files(request, course_id):
+    """
+    Pushes a Celery task which will download and compress all submission
+    files (texts, attachments) into a zip archive.
+    """
+    course_key = CourseKey.from_string(course_id)
+
+    task_api.submit_export_ora2_submission_files(request, course_key)
+
+    return JsonResponse({
+        "status": _(
+            "Attachments archive is being created. See Pending "
+            "Tasks below to follow the status of archiving process."
+        )
```

Comment:
>I don't think this message will be used directly in the UI

Actually, it's used in the UI:

![Screenshot_2020-08-06_14-51-53](https://user-images.githubusercontent.com/18251194/89529280-1fe0b180-d7f5-11ea-8950-985d2c60274e.png)

---

![Screenshot_2020-08-06_14-56-19](https://user-images.githubusercontent.com/18251194/89529286-22dba200-d7f5-11ea-90a0-58de92390824.png)

Reply:
Even so, I think it would be better not to call out the relative location. I will leave the final decision with edX but I'd favour leaving it out even if existing code does include such messaging. Simply "Attachments archive is being created.", or maybe the language can be "The task status can be tracked in the Pending Tasks section". I think this message should be handled by the frontend code in the future.